Tests/SwiftFormatTests/PrettyPrint/SelectionTests.swift

```diff
@@ -0,0 +1,359 @@
+import SwiftFormat
```

Comment:
Could you add a copyright header?

Reply:
I can. But it doesn't look like any of the test files have copyright headers. Should that just be fixed all at once (in a separate PR)?

Reply:
From what I remember, at the time we started swift-format, I think only Sources had copyright headers but not Tests in other projects, so I followed that pattern.

I'm fine with changing it, but I'd rather we do it all at once.

Reply:
Oh, sorry, then I missed that. I was just assuming that swift-format test files have copyright headers like all the other repos I work in but doesn’t seem to be the case. I would suggest that we
- Don’t add the copyright header in this PR, to be consistent
- Create some other PR to add copyright headers to all test files